help/index_en.html

```diff
@@ -13,7 +13,7 @@
 
   <center>
 
-    <p class="createdBy">Created By <a href="https://ajay.app">Ajay Ramachandran</a></p>
+    <p class="createdBy">Created By <a href="https://ajay.app">Ajay Ramachandran</a> <img src="https://ajay.app/newprofilepic.jpg" height="30" class="profilepiccircle"/></p>
```

Comment:
Since the permissions aren't set for just ajay.app (and to avoid adding extra unnecessary permissions), you may want the image to be in the repo/extension.

Reply:
Hmm, I though I changed that. I did add it to the icons folder but I guess I forgot to change the link.

Reply:
https://github.com/ajayyy/SponsorBlock/issues/218